src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -51,6 +43,12 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * @return the book with the id passed as parameter
      */
     @GetMapping("/{id}")
+    @ApiOperation(value = "Giving an Id, return the book", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved book"),
+            @ApiResponse(code = 404, message = "Book not found")
+    })
+    @ApiParam(name = "id", required = true)
```

Comment:
I think this annotation should be inside the parenthesis of the method, in the respective parameter (same for the rest of the methods) :)